B/BWA/build_tarballs.jl

```diff
@@ -17,6 +17,7 @@ script = raw"""
 cd $WORKSPACE/srcdir/bwa*/
 atomic_patch -p1 ../patches/makefile.patch
 atomic_patch -p1 ../patches/0001-Fix-building-against-GCC-10.patch
+atomic_patch -p1 ../patches/set_verbosity.patch
```

Comment:
```suggestion

# Reduce verbosity of the output to only have warnings and errors
sed -i -e 's/int bwa_verbose = 3;/int bwa_verbose = 2/' bwa.c

```
Since all you want to do is change the one value, this find/replace should do the trick without the need for a separate patch.